.github/workflows/loristest.yml

```diff
@@ -55,7 +55,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php: ['8.3', '8.4']
+        php: ['8.1', '8.2', '8.3']
```

Comment:
@kongtiaowang @driusan this looks wrong no ?

Reply:
should be 8.3 and 8.4 for the main branch

Reply:
@kongtiaowang can you please edit this file directly to the correct versions?